app/screens/shared/resource-list/ResourceListItem.spec.js

```diff
@@ -4,16 +4,16 @@ import sd from 'skin-deep';
 
```

Comment:
I just copied the old tests for this one. But your right, it's a lot nicer to have the tests done with enzyme so I made the change already in this PR.